client/components/userList.js

```diff
@@ -17,16 +13,19 @@ class UserList extends Component {
     componentDidMount() {
         fetch('http://localhost:9000/users')
             .then(response => response.json())
-            .then(data => this.setState({
-                users: data.user.map(user => {
-                    return {
-                        username: user.username,
-                        nativeLanguage: user.nativeLanguage,
-                        points: user.points,
-                        rank: user.rank
-                    }
+            .then(data => {
+                console.log(data)
+                this.setState({
+                    users: data.user.map(user => {
+                        return {
+                            username: user.username,
+                            nativeLanguage: user.nativeLanguage,
+                            points: user.points,
+                            rank: user.rank
+                        }
+                    })
                 })
-            }))
+            })
```

Comment:
@mwhitman189 

-probably better to use pluralized user/users for your data structure.
- if you don't need to filter anything out the you could probably just do `users: data.users`